### PR TITLE
Adds limits to Docker main process

### PR DESCRIPTION
**Pull request by moraesvic**

### CODE REVIEW

File: scraper/src/tests/memory_test/fill_memory.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(int argc, char * argv[])
```
Comment (MarcosFleury) [inline scraper/src/tests/memory_test/fill_memory.c:4]: Do you think we need to have this test and its binary here?

File: scraper/src/tests/memory_test/report_2022-02-25.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+LOWER BOUND (MB)  UPPER BOUND (MB)     COUNT       (%)
```
Comment (MarcosFleury) [inline scraper/src/tests/memory_test/report_2022-02-25.txt:1]: Do we need to have this file here?

**Review by MarcosFleury — APPROVED** [review]
I think the binary and the txt file from the memory test don't need to be pushed, what do you think?

**Comment by moraesvic** [comment]
I think the source for the test can be kept, we might need to use it again later, but the binary and the report can be removed. Actually, I think I will move the report to the `docs` folder,

**Review by MarcosFleury — APPROVED** [review]

**Pull request merged**